application-provider/src/main/java/org/jboss/intersmash/applications/ApplicationProvider.java

```diff
@@ -31,6 +31,7 @@ public class ApplicationProvider {
 	static final String WILDFLY_ELYTRON_OIDC_CLIENT_KEYCLOAK_DEPLOYMENT = "wildfly-elytron-oidc-client-keycloak";
 	static final String WILDFLY_WEB_CACHE_OFFLOAD_INFINISPAN_DEPLOYMENT = "wildfly-web-cache-offload-infinispan";
 	static final String WILDFLY_DISTRIBUTED_SESSIONS_INFINISPAN_DEPLOYMENT = "wildfly-distributed-sessions-infinispan";
+	static final String WILDFLY_AMQ_BROKER_SSL_DEPLOYMENT = "wildfly-amq-broker-ssl";
```

Comment:
Could it be s/WILDFLY_AMQ_BROKER_SSL_DEPLOYMENT/WILDFLY_ACTIVEMQ_ARTEMIS_SSL_DEPLOYMENT/ ?